src/Bookings.js

```diff
@@ -1,17 +1,24 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
 import SearchResults from "./SearchResults.js";
-import FakeBookings from "./data/fakeBookings.json";
+import "./App.css";
 
 const Bookings = () => {
+  useEffect(() => {
+    console.log("Hi, this is the first using useEfct");
+    fetch("https://cyf-react.glitch.me")
+      .then(res => res.json())
+      .then(data => setBookings(data));
+  }, []);
+  const [bookings, setBookings] = useState([]);
```

Comment:
Nice work! There's one little thing that I would suggest changing to follow the *convention* of React Hooks. We normally put all of our `useState` calls at the top of the component:

```js
  const [bookings, setBookings] = useState([]);

  useEffect(() => {
    console.log("Hi, this is the first using useEfct");
    fetch("https://cyf-react.glitch.me")
      .then(res => res.json())
      .then(data => setBookings(data));
  }, []);
```